commands/support.js

```diff
@@ -128,6 +128,24 @@ module.exports.init = controller => {
             'Please see the following link for tips to get your Swing  Wallet working again! https://goo.gl/MhMZpz'
         )
     })
+    
+
+    controller.hears(['!social'], 'ambient,bot_message,direct_message,direct_mention,mention', (bot, message) => {
+        bot.reply(
+            message,
+            '*Follow BTCZ:*\n' +
+            'Facebook - https://www.facebook.com/BTCZCommunity\n' +
+            'Twitter - https://twitter.com/BitcoinZTeam\n' +
+            'Youtube - https://www.youtube.com/channel/UC2MBhoVK45Eqp2cMb2bFuUA\n' +
+            'Instagram- https://slack.btcz.rocks/\n' +
```

Comment:
dont know the instagram link but the link here is a duplicate for Slack invite link